src/getchar.c

```diff
@@ -1371,8 +1374,16 @@ gotchars(char_u *chars, int len)
 
 	// Handle one byte at a time; no translation to be done.
 	for (i = 0; i < state.buflen; ++i)
+        {
 	    updatescript(state.buf[i]);
 
+	    if (typedchars_pos < MAXMAPLEN)
+	    {
+		typedchars[typedchars_pos] = state.buf[i];
+		typedchars_pos++;
+	    }
```

Comment:
You need to call `vim_unescape_csi()` on `state.buf[]`, otherwise chars like "…" will have K_SPECIAL/CSI escaped.